Artifacts/windows-powershellcore/installPowerShellCore.ps1

```diff
@@ -0,0 +1,79 @@
+﻿param
```

Comment:
Consider adding `[CmdletBinding()]` to support common parameters like `-Verbose` to automatically pass on to other PS calls.